flyway-core/src/test/resources/migration/dbsupport/sqlserver/sql/assembly/FlywayTestAssembly.cs

```diff
@@ -0,0 +1,173 @@
+/**
```

Comment:
The CLR assembly source code. Must be manually compiled and included in the test script as an hexadecimal string (see generate_assembly.txt).